### ISSUE TITLE
[Bug]: Failed to download binary for Laravel extension: php-parser-v0.1.34-arm64-darwin

### ISSUE THREAD

**Issue by Adams-Ijachi** [laravel/vs-code-extension#126]
### Extension Version

0.1.14

### PHP Binary

Herd

### Operating System

macOS

### What happened?

Tried installing the extension and keep getting `Failed to download binary for Laravel extension: php-parser-v0.1.34-arm64-darwin`

M1 apple silicon

**Comment by hussain4real** [comment]
are you using vscode-insider?

**Comment by Adams-Ijachi** [comment]
no i am not just plain vscode @hussain4real

**Comment by hussain4real** [comment]
You might have to do clean uninstall of vscode:
[Video Tutorial](https://youtu.be/Hf4kcNN91-k?si=RERXYBJ159LRpPFx)

**Comment by mangtri99** [comment]
same issue here with M2 Air

**Comment by selase** [comment]
> Video Tutorial

Yes, I am using VSCode Insider and having the same issues. What do you recommend?

**Comment by hussain4real** [comment]
> > Video Tutorial
> 
> Yes, I am using VSCode Insider and having the same issues. What do you recommend?

you could make a clean install of vscode

**Comment by francoism90** [comment]
Same issue for me: Failed to download binary for Laravel extension: php-parser-v0.1.34-arm64-linux

I'm running this inside a devcontainer on macOS with a M1 Pro.

**Comment by TitasGailius** [comment]
Thanks for reporting this. We’re clearing out some older issues that may have been fixed by the latest improvements in the extension. Feel free to reopen if you’re still experiencing anything similar.
